libcontainerd/container_windows.go

```diff
@@ -41,6 +41,14 @@ func (ctr *container) start() error {
 		return err
 	}
 
+	if ctr.ociSpec.Windows.Servicing {
+		if err := hcsshim.ShutdownComputeSystem(ctr.containerID, hcsshim.TimeoutInfinite, ""); err != nil {
+			logrus.Errorf("Failed during cleanup of servicing container: %s", err)
+			return err
+		}
+		return nil
```

Comment:
Can you add an explicit comment here that we are exiting the `start()` call early at this point regardless, and not going on to create any processes in the container. (I got confused reading the change initially and didn't realise that at first.)